src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+  let current;
+
+  for (let i = 0; i < propertyValues.length; i++) {
+    current = propertyValues[i];
+
+    for (let j = i + 1; j < propertyValues.length; j++) {
+      if (current === propertyValues[j]) {
+        return null;
+      }
+    }
+  }
+
+  for (const key in robot) {
+    inverse[robot[key]] = key;
+  }
```

Comment:
In this loop, you are inverting the object. However, you should also check for duplicates here. If the 'inverse' object already has a property with the current value, it means that the value is duplicated in the original object. In this case, you should return null.